src/components/shareable-url-modal/published-view/published-view.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classnames from 'classnames';
+import UrlBox from '../url-box/url-box';
+import Button from '../../ui/button';
+import Dropdown from '../../ui/dropdown';
+import MenuOption from '../../ui/menu-option';
+
+import { getFilteredPlatforms, handleResponseUrl } from '../utils';
+
+const PublishedView = ({
+  hostingPlatformLocalStorageVal,
+  hostingPlatforms,
+  onChange,
+  onCopyClick,
+  onRepublishClick,
+  platform,
+  showCopied,
+}) => {
+  const platformsKeys = Object.keys(hostingPlatformLocalStorageVal);
+  const platformsVal = Object.values(hostingPlatformLocalStorageVal);
+
+  const url = platform
+    ? hostingPlatformLocalStorageVal[platform]['endpoint']
+    : platformsVal[0]['endpoint'];
+
+  const filteredPlatforms = getFilteredPlatforms(
+    hostingPlatforms,
+    platformsKeys
+  );
+
+  const href = handleResponseUrl(url, platform || platformsVal[0]['platform']);
+
+  return (
+    <>
+      <div className="shareable-url-modal__published">
+        <div className="shareable-url-modal__content-title">
+          Publish and Share Kedro-Viz
+        </div>
+        {platformsKeys.length === 1 ? (
```

Comment:
I have published once on AWS and the next time, I would like to publish on Azure. Now, my localStorage contains 1 key which makes this condition true. How can I publish on Azure ? I do not see a link on `publish and host Kedro-Viz with a new link.` . Am I missing something ?

``` <div className="shareable-url-modal__published-action">
        <p className="shareable-url-modal__published-action-text">
          Republish Kedro-Viz to push new updates,
          <br />
          or publish and host Kedro-Viz with a new link.
        </p>
        <Button mode="secondary" onClick={onRepublishClick} size="small">
          Republish
        </Button>
      </div>
  ```